docs/AboutUs.adoc

```diff
@@ -12,7 +12,7 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 
 === Rahul
 image::rrtheonlyone.png[width="150", align="left"]
-{empty}[https://github.com/rrtheonlyone[github]] [https://www.linkedin.com/in/rahul-rajesh-979919110/[portfolio]]
+{empty}[https://github.com/rrtheonlyone[github]] [<<rahulrajesh#, portfolio>>]
```

Comment:
Your PPP file name should be your github username

Reply:
Done!